modules/table/src/main/java/org/apache/ignite/internal/table/InternalTable.java

```diff
@@ -246,7 +246,7 @@ CompletableFuture<List<BinaryRow>> getAll(
      * @param tx   The transaction.
      * @return Future representing pending completion of the operation.
      */
-    CompletableFuture<Collection<BinaryRow>> deleteAll(Collection<BinaryRowEx> rows, @Nullable InternalTransaction tx);
+    CompletableFuture<List<BinaryRow>> deleteAll(Collection<BinaryRowEx> rows, @Nullable InternalTransaction tx);
```

Comment:
It is necessary to change the documentation and describe what will be returned in the future with examples.